asciidoc/volume3/tf3-ch-8.3.2-biceps-content.adoc

```diff
@@ -146,6 +146,55 @@ In addition to specific device specializations (see <<vol3_clause_device_special
 #TODO:  ADD TERM CODE IDENTIFIERS FOR THE SYSTEM TYPES IN THE TABLE ABOVE + SEE ADDITIONAL TYPES AT https://confluence.hl7.org/pages/viewpage.action?pageId=86972238[ToI "What is a Central?"]#
 ////
 
+[#vol3_clause_globally_unique_code_system_id]
+===== Globally Unique Vendor-specific Coding System Identifier
+
+The <<ref_ieee_11073_10101_2020>> nomenclature standard gives medical device vendors the freedom to define their own codes for new concepts which are currently not part of the standard nomenclature.
+
+The <<ref_ieee_11073_10700_2022>> Base Requirements for Participants in a Service-Oriented Device Connectivity (SDC) System defines two requirements for use of private codes as detailed in <<ref_ieee_11073_10101_2020>>:
+
+* *TR1357*: If there is no standardized code to convey semantics, an SDC BASE PARTICIPANT S-H-O-U-L-D populate pm:CodedValue with a private code as detailed in [IEEE 11073-10101].
```

Comment:
Why are there dashes in the keyword?

Reply:
I wanted to quote the original Base-PKP requirement. In order to avoid any complaints from the build pipeline checker (requirement key words in a text outside of an requirement) I used dashes to cheat the checker :-) Any better ideas?

Reply:
Yikes! That should not end up in a broken build pipeline. I am going to take care of that. For now, we leave it that way and I am going to fix this later (while I am at it, see #243).